plugins/BEdita/Core/tests/TestCase/Model/Entity/StreamTest.php

```diff
@@ -339,8 +346,7 @@ public function testGetUrlPrivate()
 
     /**
      * Read data from image if is possible
-     *
-     * @param Stream $stream stream entity
+     * @param EntityStream $stream stream entity
```

Comment:
This change MUST be reverted in order to work with the latest PHP CS version:
```suggestion
     *
     * @param \BEdita\Core\Model\Entity\Stream $stream Stream entity.
```